tcms/testruns/templates/testruns/get.html

```diff
@@ -119,6 +119,9 @@ <h2 class="card-pf-title" style="text-align: left">
 
 <script src="{% static 'typeahead.js/dist/typeahead.jquery.min.js' %}"></script>
 <script src="{% static 'bootstrap-switch/dist/js/bootstrap-switch.min.js' %}"></script>
+<script src="{% static 'moment/moment.js' %}"></script>
+<script src="{% static 'moment-timezone/moment-timezone.js' %}"></script>
```

Comment:
min too

Reply:
```
$ ls node_modules/moment-timezone/
LICENSE                  builds                   composer.json            index.js                 moment-timezone.js
README.md                changelog.md             data                     moment-timezone-utils.js package.json
```
there is no `min` folder in `moment-timezone`.

OTOH it looks like we don't need this file at all, because the same functions are loaded with `moment-timezone/builds/moment-timezone-with-data.min.js`. Maybe this caused the problem you were experience. 

I deleted it, so try again now. On my side, everything is still working as expected.

Reply:
> there is no min folder in moment-timezone.

The location is up to the packager. The file is at `node_modules/moment-timezone/builds/moment-timezone.min.js`.

> Maybe this caused the problem you were experience.

yes, it did.